cli/blobconverter/__init__.py

```diff
@@ -95,7 +96,7 @@ def build(self):
 
 
 __defaults = {
-    "url": "https://blobconverter.luxonis.com",
+    "url": "https://dev-blobconverter.luxonis.com",
```

Comment:
Things like defaults here should fall to https://blobconverter not the dev one.

Might make sense to create a separate branch "rvc3-merge" and refactor those, then make a PR?

Reply:
Alright, will do